app/iot/devices.py

```diff
@@ -8,57 +8,57 @@
 
 # of course this code looks dumb, but imagine some real implementations of each method here
 class HueLightDevice:
-    def connect(self) -> None:
+    async def connect(self) -> None:
         print("Connecting Hue Light.")
-        time.sleep(TIME_TO_SLEEP)
+        await asyncio.sleep(TIME_TO_SLEEP)
         print("Hue Light connected.")
 
     def disconnect(self) -> None:
```

Comment:
`Make your code use asyncio - connect/disconnect/send_message must be async functions`